selfdrive/car/interfaces.py

```diff
@@ -101,6 +103,19 @@ def get_steer_feedforward_default(desired_angle, v_ego):
   def get_steer_feedforward_function(self):
     return self.get_steer_feedforward_default
 
+  @staticmethod
+  def convert_latacc_to_torque_linear(error, pid_lat_accel, lateral_accel_deadzone, live_torque_params, v_ego):
```

Comment:
This isn't a great name, something like torque_from_lateral_accel_linear, is as long but much clearer